spring-cloud-sleuth-core/src/main/java/org/springframework/cloud/sleuth/Tracer.java

```diff
@@ -19,30 +19,40 @@
 import java.util.concurrent.Callable;
 
 /**
- * The TraceManager class is the primary way for instrumentation code (note user code) to
- * interact with the library. It provides methods to create and manipulate spans.
+ * <p>
+ *     The TraceManager class is the primary way for instrumentation code (note user code) to
+ *     interact with the library. It provides methods to create and manipulate spans.
+ * </p>
  *
- * A 'Span' represents a length of time. It has many other attributes such as a name, ID,
- * and even potentially a set of key/value strings attached to it.
+ * <p>
+ *     A 'span' represents a length of time. It has many other attributes such as a name, ID,
+ *     and even potentially a set of key/value strings attached to it.
+ * </p>
  *
- * Each thread in your application has a single currently active currentSpan associated
- * with it. When this is non-null, it represents the current operation that the thread is
- * doing. Spans are NOT thread-safe, and must never be used by multiple threads at once.
- * With care, it is possible to safely pass a Span object between threads, but in most
- * cases this is not necessary.
+ * <p>
+ *     Each thread in your application has a single currently active currentSpan associated
+ *     with it. When this is non-null, it represents the current operation that the thread is
+ *     doing. spans are NOT thread-safe, and must never be used by multiple threads at once.
+ *     With care, it is possible to safely pass a span object between threads, but in most
+ *     cases this is not necessary.
+ * </p>
  *
- * The 'startTrace' method in this class starts a new span.
+ * <p>The {@code startTrace} method in this class starts a new span.</p>
+ * <p>The {@code joinTrace} method creates a new span which has this thread's currentSpan as one of its parents</p>
  *
- * <li>Create a TraceSpan object to manage the new Span.</li>
- * </ul>
- *
- * The 'joinTrace' method creates a new Span which has this thread's currentSpan as one of its parents
+ * <p>
+ *     The {@code continueSpan} method creates a new physical span that logically is a continuation of the provided span.
```

Comment:
keep multi-line comments about a method on the method itself. If overview, try to limit to one sentence of duplication, as this text can easily drift between here and the methods themselves